### DISCUSSION TITLE
Dear all, is it possible to add on a simply way alternate colors for odd / even rows?

### DISCUSSION THREAD

**Discussion by itebet** [microsoft-search/pnp-modern-search#3653] [Q&A]
Adding alternate row colors for odd / even rows (like white / grey) would be really useful for long list format results.
If you know some pieces of code to all will be appreciated. 
If you can add it as a layout option on the UI it will be even better.
Thank you in advance.

**Comment by kasperbolarsen** [comment]
Sure, I assume you are referring to the Detailed List layout? It is absolutely possible to customize that layout with some CSS that will make it look like this, https://youtu.be/iJm93QtcFMc?si=s8M2YxiSG9I8qUJA

Please add the customized layout to https://github.com/microsoft-search/pnp-modern-search-layouts in order for other to use the template 
#SharingIsCaring

**Comment by itebet** [comment]
Hi, I'm sorry but I'm not a developer, from the video you're pointing I see that he uses the expression {{if(@rowIndex%2==0,'ms-bgColor-themeLight','')"}} but I inserted on Detailed List layout with no effects.
I also tried to modify the CSS adding background-color property with "nth-child(odd)" to several page styles, but it's just ignoring it.
Any other ideas? Looks too complicated without being a developer.

**Reply by kasperbolarsen** [reply]
It is pretty straightforward for a dev, but most likely complete gibberish for a non dev.

As we are all volunteers the best I can offer is to convert this request to a "enhancement issue", and if you are lucky a dev might find the issue interesting ( or need the feature him/her self) and implement it.  This is no guarantee that it will be implemented. 
If you need that feature you might partner up with a dev from your own organization or with a freelance dev, and have they implement it.

**Reply by itebet** [reply]
Thanks a lot, that will be really appreciated.
Best regards.

**Comment by kasperbolarsen** [comment]
converted to an issue
